README.md

```diff
@@ -104,7 +104,7 @@ The following constants are reported as the only parameter to the
 
 ### Methods
 
-- `media.getCurrentAmplitude`: Returns the current position within an audio file.
+- `media.getCurrentAmplitude`: Returns the current amplitude of the current recording.
```

Comment:
Why? I only copied from the next part as you did with all the other definitions :S